src/main/kotlin/uk/gov/justice/digital/hmpps/approvedpremisesapi/service/UserService.kt

```diff
@@ -49,7 +49,9 @@ class UserService(
   private val log = LoggerFactory.getLogger(this::class.java)
 
   fun getUsersByPartialName(name: String): List<UserEntity> {
-    return userRepository.findByNameContainingIgnoreCase(name)
+    return userRepository.findByNameContainingIgnoreCase(name).filter {
+      it.isActive == true
+    }
```

Comment:
Can we update this so the filtering is done on the db side? We'll probably need to paginate this stuff eventually, so we'll need to get a consistent number of results for each page.